php/hleb2-swoole/public/index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Swoole\Http\Request;
+use Swoole\Http\Response;
+use Swoole\Http\Server;
+
+include __DIR__ . "/../vendor/autoload.php";
+
+$http = new Server('0.0.0.0', 3000, SWOOLE_BASE, SWOOLE_SOCK_TCP);
+$http->set([
+    'worker_num' => swoole_cpu_num() * 2,
```

Comment:
is there a way to use a swoole config file for those options ? in order to make sure we use the same file for all swoole based

Reply:
This is the most common type of Swoole configuration I've seen so far. It's unlikely that you'll be able to preconfigure Swoole at the Docker installation stage, as this configuration is tied to the application's runtime behavior.

Reply:
Yes, sure. I just meant if swoole could read an external file to be configured, an ini, XML or else

Reply:
> Yes, sure. I just meant if swoole could read an external file to be configured, an ini, XML or else

I completely support the idea of making configurations unified and editable in a single place, and I’m genuinely impressed with the concept and implementation of the benchmarks themselves. However, I don’t currently have any ideas on how to make the Swoole configuration fully universal. Frameworks like Laravel, for example, might still require additional adjustments due to their specific requirements. Even if we imagine manually generating identical YAML files for Swoole settings and converting each into global PHP variables to set in the configuration, differences would likely persist regardless.